FileAPI/historical.https.html

```diff
@@ -53,9 +53,13 @@
         assert_false('isClosed' in Blob.prototype, 'isClosed in Blob.prototype');
     }, 'Blob.close() should not be supported');
 
-    // Only add service worker test if service workers are actually supported.
-    if (navigator.serviceWorker)
```

Comment:
@annevk Was the removal of the condition here accidental? It causes this test to fail in implementations without Service Workers, such as jsdom.

Reply:
Yeah, we shouldn't have conditional tests. That makes comparisons between implementations fragile. I'd be okay with splitting this test out, but these kind of conditionals in tests are bad.

Reply:
But now that means this test will fail on implementations that correctly don't implement what the test is trying to make sure they don't implement, just because they don't implement something else as well? That does seem weird to me...
At least that was my original reasoning for having the conditional here.